runner/components/io.siddhi.distribution.common/src/main/java/io/siddhi/distribution/common/common/utils/config/FileConfigManager.java

```diff
@@ -42,9 +42,9 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(FileConfigManager.class);
 
     private ConfigProvider configProvider;
-    private List<Extension> extensions;
-    private List<Reference> references;
-    private Map<String, String> properties;
+    private List<Extension> extensions = new ArrayList<>();
```

Comment:
@niveathika I have updated FileConfigManager to initialize it's references to avoid NPE if init() is not called.
Please check.

Reply:
@pcnfernando Let's lazy load the configs instead of mandating calling init() method. In the above way, user do not know init() method needs to be called and this can return empty arrays for valid configs

Reply:
it has already been handled in [commit](https://github.com/siddhi-io/distribution/pull/273/commits/d8bbb74f1665ed96f68f285f4af10fbf1a1a81df)

Reply:
@pcnfernando Ack